subsys/bluetooth/mesh/scheduler_srv.c

```diff
@@ -639,6 +639,16 @@ static int action_set(const struct bt_mesh_model *model, struct bt_mesh_msg_ctx
 		run_scheduler(srv);
 	}
 
+	if ((srv->sch_reg[idx].action == BT_MESH_SCHEDULER_NO_ACTIONS) &&
+	    (srv->active_bitmap & BIT(idx))) {
```

Comment:
```suggestion
	    IS_BIT_SET(srv->active_bitmap, idx)) {
```
just to be pedantic :)

Reply:
Where is IS_BIT_SET defined? (Or do you want me to define it in the file?)

Reply:
No, we should use Zephyr API for this: https://github.com/zephyrproject-rtos/zephyr/blob/main/include/zephyr/sys/util_macro.h#L96-L102

Reply:
That is not upmerged yet :(

Reply:
I see this is not upmerged yet :) I'm so quick with it. Then let's leave `as is`